### PR TITLE
[Azure RDMA] [Doc] Update tutorial of rdma for admin to enable it in OpenPAI.

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

File: docs/pai-management/doc/azure/enable-az-rdma.md [hunk 1]
```diff
@@ -69,4 +69,32 @@ sudo ./paictl.py utility ssh -p /path/to/cluster/config -f rdma=true -c "sudo /b
...
+#    # HDFS, zookeeper data path on your cluster machine.
+#    data-path: "/datastorage"
+#
+#    # Enable QoS feature or not. Default value is "true"
+#    qos-switch: "true"
+#
+#    # If your cluster is created by Azure and the machine is rdma enabled.
+#    # Set this configuration as  "true", the rdma environment will be set into your container.
+    az-rdma: "true"
+```
```
Comment (hao1939) [inline docs/pai-management/doc/azure/enable-az-rdma.md]: what's the usage of file ```cluster.common.az-rdma``` and set its value as ```"true""```?

Comment (ydye) [reply docs/pai-management/doc/azure/enable-az-rdma.md]: Grant  the ability of making container rdma capable to cluster.

Comment (ydye) [reply docs/pai-management/doc/azure/enable-az-rdma.md]: Default value is "false".

Comment (ydye) [reply docs/pai-management/doc/azure/enable-az-rdma.md]: Here is an example

**Pull request merged**